engine/XcodeUnitTestEngine.php

```diff
@@ -56,7 +56,10 @@ private function shouldGenerateCoverage() {
     return ($arcCoverageFlag !== false) && $this->hasCoverageKey;
   }
 
-  protected function loadEnvironment() {
+  private function checkArcConfig() {
```

Comment:
Did you consider naming this to `getArcConfig`? The `get` prefix is more consistent with arcanist's other method calls (e.g. `getConfigurationManager`).